lifetimes/estimation.py

```diff
@@ -37,6 +37,117 @@ def _print_params(self):
         return s.strip(', ')
 
 
+class GammaGammaFitter(BaseFitter):
+    def __init__(self, penalizer_coef=0.):
+        self.penalizer_coef = penalizer_coef
+
+    @staticmethod
+    def _negative_log_likelihood(params, frequency, avg_monetary_value, penalizer_coef):
+        if any(i < 0 for i in params):
+            return np.inf
+
+        p, q, v = params
+
+        x = frequency.astype('int')
+        m = avg_monetary_value.astype('int')
+
+        p_0 = np.log(special.gamma(p*x+q)) - np.log(special.gamma(p)) - np.log(special.gamma(x)) - np.log(special.gamma(q))
```

Comment:
Also, using `gammaln(*)` in scipy: http://docs.scipy.org/doc/scipy/reference/generated/scipy.special.gammaln.html is more stable than `np.log(special.gamma(*))`